### PR TITLE
tested roles

### PR DESCRIPTION
**Pull request by MrishoLukamba**
Check out the comments which i left

### CODE REVIEW

File: pallets/roles/src/tests.rs [hunk 1]
```diff
@@ -1,9 +1,115 @@
...
+						account_id:1,
+						nft_index:Vec::new(),
+						age: System::block_number(),
+					}
+				],
+				vec![]
+			)
+		);
+		//---house seller should fail successfully----
+		/*assert_ne!(WaitingList::<Test>::get(),
```
Comment (ndkazu) [inline pallets/roles/src/tests.rs:44]: For "success on failure" tests you should try `assert_noop`. see the test in housing_fund for reference.

File: pallets/roles/src/tests.rs [hunk 2]
```diff
@@ -1,9 +1,115 @@
...
+		/*assert_eq!(TenantLog::<Test>::get(1),
+			Some(
+				Tenant{
+					account_id:1,
+					rent:0,
+					age: System::block_number(),
+				}
+			)
+		)*/
+		//This test is failing because There is no an update on Tenant Storage when initializing
```
Comment (ndkazu) [inline pallets/roles/src/tests.rs:76]: Well done! I will correct this today.

File: pallets/roles/src/tests.rs [hunk 3]
```diff
@@ -1,9 +1,115 @@
...
+
+}
+
+#[test]
+fn test_dispatchable_calls(){
+	new_test_ext().execute_with(|| {
+		//----testing account approval-----
+		//let call = Box::new(Call::RoleModule(RoleCall::account_approval{account:2}));
+		//assert_ok!(Sudo::sudo(Origin::signed(1),call));
+		//assert_ok!(RoleModule::account_approval(RawOrigin::root(),2));
```
Comment (ndkazu) [inline pallets/roles/src/tests.rs:109]: I think it is `RawOrigin::Root()` and not `RawOrigin::root()`. notice the capital 'R' in root.

**Review by ndkazu — CHANGES_REQUESTED** [review]

File: pallets/roles/src/tests.rs (hunk 3 above)
Comment (MrishoLukamba) [reply pallets/roles/src/tests.rs:109]: well Root is not present its only root

File: pallets/roles/src/tests.rs (hunk 1 above)
Comment (MrishoLukamba) [reply pallets/roles/src/tests.rs:44]: assert_noop doesnt work either maybe you can try it

**Pull request merged**